working-together/README.md

```diff
@@ -27,17 +27,60 @@ A guide for working well together.
 - Ask for Feedback (What did we do well? What could we have done better? It’ll
   feel weird, do it anyway)
 
-## Meeting Guidelines
+## Remote Meeting Guidelines
 
 We've compiled some guidelines from various sources to help people feel included
-and respected during meetings. You can [download a double-sided printout] of
+and valued during remote meetings that happen over video conferencing software.
+
+- **Time zones:** Consider time zones when scheduling meetings - is the meeting 
+    time respectful of everyone's work schedule?
+- **Be Courteous::** Be respectful of others on your team.
+  - Mute yourself when not talking to minimize background noise.
+  - Ensure everyone on your team can hear and understand you clearly.
```

Comment:
```suggestion
  - Confirm that everyone on your team can hear and understand you clearly.
```


I wasn't sure how I would be able to ensure this for everyone else, so I made a suggested change to wording here, what do you think?

Reply:
I think this change didn't make it in? Am I understanding the intent behind this sentence correctly with my suggestion? I feel like putting the onus on one person to ensure something for others is challenging.